rosrust/Cargo.toml

```diff
@@ -15,7 +15,7 @@ error-chain = "0.12.1"
 lazy_static = "1.4.0"
 log = "0.4.8"
 nix = "0.15.0"
-rosrust_codegen = "0.9.2"
+rosrust_codegen = 0.9.3
```

Comment:
I'm just some random person, but this should still be in quotes.

Reply:
> I'm just some random person, but this should still be in quotes.

thanks